checkout.js

```diff
@@ -1,13 +1,7 @@
-
 function selectSaleItems(items) {
-
   return items.discount > 0
 }
 
 const printSaleItems = (items) => items.filter(selectSaleItems)
```

Comment:
This works but you didn't need the new function. `selectSaleItems` is already being exported and called by the tests.

```
function selectSaleItems(items) {
    return items.filter((item) => item.discount > 0)
}
```